recipe/meta.yaml

```diff
@@ -1,64 +1,63 @@
-{% set name = "gpstime" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.4" %}
 
 package:
-  name: "{{ name }}"
-  version: "{{ version }}"
+  name: gpstime
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: dd26382df3ded8fb203f1ae8c77b3eaee6b5918c4d1bbe4e49721725f11413c8
+  url: https://git.ligo.org/cds/gpstime/-/archive/{{ version }}/gpstime-{{ version }}.tar.gz
+  sha256: 5908069f99884083881aee663a6c9bbb7b87a67b768d61eb299a04dd1c2db091
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
-  entry_points:
-    - gpstime = gpstime.__main__:main
-    - ietf-leap-seconds = ietf_leap_seconds:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
```

Comment:
Since this version now uses the `scripts` keyword in `setuptools.setup()`, and doesn't use `entry_points`, this recipe is no longer applicable as `noarch: python`. So, you will need to remove that key and add `skip: true  # [win]` if needed.

**OR** we can leave the `noarch: python` bit in and use the following patch in the conda build _only_ to continue using `entry_points` here (since, as we have discussed, conda-built entry points don't use `pkg_resources` so don't suffer from the same performance problems):

```patch
diff --git a/setup.py b/setup.py
index 80accaa..130a670 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,11 @@ setup(
     ],

     packages=['gpstime'],
-    scripts=['bin/gpstime'],
+    entry_points={
+        'console_scripts': [
+            'gpstime = gpstime.__main__:main',
+        ],
+    },

     tests_require=[
         'python-dateutil',
```